client/src/pages/onboarding/OnboardingPage.tsx

```diff
@@ -41,7 +41,6 @@ export const OnboardingPage: React.FC = () => {
       dispatch(fetchAllUseCasesByCharId(currentCharacter.id))
       navigate('/dashboard')
     } else {
-      dispatch({ type: 'demo/RESET' })
```

Comment:
This is put here so that when users refresh the page halfway through the onboarding the state is reset. This is because the user might come back at a later time and we don't know if the user still has a wallet ready (they might have deleted it from their phone). So we just assume the user want's to start over.

I think we can solve this by creating a `demo/resetState` that resets the whole state except the configuration slice, and a `demo/reset` that resets all slices. The `demo/resetState` should replace all the current resets and the `demo/reset` should be an option in the CMD+K menu.

Does this make sense :)? If you have any ideas on solving it in another way lmk!

Reply:
Okay. But I've removed the configuration slice and moved the configuration logic to connection & credential slices, so that would be connection slice and credential slice